libs/application/types/src/lib/Fields.ts

```diff
@@ -69,15 +69,36 @@ export type TableRepeaterItem = {
   displayInTable?: boolean
   label?: StaticText
   placeholder?: StaticText
-  options?: { label: StaticText; value: string }[]
+  options?:
+    | {
+        label: StaticText
+        value: string
+        tooltip?: StaticText
+      }[]
+    | ((
+        application: Application,
+        activeField: Record<string, string>,
+      ) =>
+        | {
+            label: StaticText
+            value: string
+            tooltip?: StaticText
+          }[]
+        | undefined)
```

Comment:
Can this function return an empty array to skip the possibility to return undefined?